message/abandon.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	servicebus "github.com/Azure/azure-service-bus-go"
+	"github.com/devigned/tab"
```

Comment:
mention service bus is using this as well which is really nice since you can plugin your logger for both.

Reply:
mentioned in comments